src/main.js

```diff
@@ -4,14 +4,95 @@ import { spawn } from 'child_process'
 import parser from 'xml2json'
 import openAboutWindow from 'about-window'
 
-import browsers from './browsers'
+// import browsers from './browsers'
+
+import memFs from 'mem-fs'
+import editor from 'mem-fs-editor'
+
+class Notification {
+  constructor(type, msg, disposed = false) {
+    this.type = type
+    this.msg = msg
+    this.disposed = disposed
+  }
+
+  setDisposed(val) {
+    this.disposed = val
+  }
+
+  dispose() {
+    this.disposed = true
+  }
+}
 
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
 let pickerWindow = null
 let tray = null
 let appIsReady = false
 
+let configFileName = '.config/browserosaurus.json'
+
+let configDefault = {}
+let configUser = {}
+let notifications = []
+var version = app.getVersion()
+
+const loadConfig = () => {
+  return new Promise((fulfill, reject) => {
+    var configPath = require('os').homedir() + '/' + configFileName
```

Comment:
As we'll only ever support Mac, is `~` easier here?

Reply:
Actually tilde expansion is a pretty complex feature of shell and not the OS. I'm not sure how node handles it but I'm sure that ```var configPath = '~/' + configFileName ``` doesn't work. It was my first attempt.

Reply:
ah, ok. Fair enough.